modules/ROOT/pages/message-logging-policy.adoc

```diff
@@ -63,7 +64,27 @@ Error handler after API flow:
 
 image::apim-message-logging-policy-states-3.png[State after the error handler executes]
 
-== Logging Policy in XML
+
+== Configuring Policy Parameters
+
+When you apply the Message Logging policy to your API from the UI, you can configure the following parameters:
+
+image::apim-message-logging-policy-ui.png[Message Logging Policy UI]
+
+[%header%autowidth.spread]
+|===
+|Parameter |Description
+|Message |A DataWeave expression that's transformed into a message to be logged.
+Note: If the payload is used as part of the message log, do not configure the Listener as non-repeatable (by default this setting is off).
+|Conditional |The condition expressed as a DataWeave expression that returns true or false that needs to be satisfied to log the message. This field is optional.
+|Category |The category that the log has. This parameter is optional. If no category is set, the default is the `org.mule.runtime.logging.policy-<policy id>` value.
+|Level |Defines if the message is logged according to the configuration in your log4j2 file.
+|Before Calling API |Set to cause your logging policy to be applied before the API is called.
```

Comment:
Partially true statement. I would say something like the following, instead: "Set to cause your logging policy to log messages before the API endpoint is called, taking into account the order in which this policy is applied".

Reply:
Done! Thanks!